### PR TITLE
[TSLint Rule] - adding comment-format check-space rule

### PR DESCRIPTION
**Pull request by bluong**
This checks if there is a single space in a single line comment between the / and the comment itself

Neat:
```typescript
// Insert comment here
```

Messy:
```typescript
//Insert Comment here
```

### CODE REVIEW

File: plottable.d.ts [hunk 1]
```diff
@@ -2171,6 +2171,7 @@ declare module Plottable {
     }
 }
 
+
```
Comment (aicioara) [inline plottable.d.ts]: This is a bit weird. Could you double check that the compilation actually yielded that?

File: src/components/axes/numericAxis.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-//<reference path="../../reference.ts" />
+///<reference path="../../reference.ts" />
```
Comment (aicioara) [inline src/components/axes/numericAxis.ts]: This looked pretty serious. Is this caught by the linter as well? If not, it should

**Comment by aicioara** [comment]
I've seen you opened quite a few tickets, so just not to forget, in case this is not covered yet:
- Comments should be enforced to always begin with either Capital letter or lower letter (we have mixture)
- I suggest we should not have inline comments like this (but up to you)
```typescript
doSomethin(); // comment on same line
```

File: plottable.d.ts (hunk 1 above)
Comment (bluong) [inline plottable.d.ts]: This probably happened due to the numericAxis.ts change

File: src/components/axes/numericAxis.ts (hunk 2 above)
Comment (bluong) [inline src/components/axes/numericAxis.ts]: I am unsure how to tell the linter to look for the above.  The above seems very specific to Plottable

Comment (jtlan) [inline src/components/axes/numericAxis.ts]: Not really, this seems to be the way Typescript recommends doing this:

http://www.typescriptlang.org/Handbook#modules-splitting-across-files

**Pull request merged**